src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -137,6 +138,94 @@ public class Utils {
 
   private static final KCLogger LOGGER = new KCLogger(Utils.class.getName());
 
+  private static final Pattern unquotedIdentifierPattern =
+      Pattern.compile("[$][0-9]+|[_a-zA-Z][_a-zA-Z0-9]*[$]?[_a-zA-Z0-9]*");
+
+  private static final String[] reservedKeywords = {
```

Comment:
Do we need to put these reservedKeywords here? It will not always be kept in sync / be the full ist